gsdk/src/signer/mod.rs

```diff
@@ -37,7 +35,7 @@ mod utils;
 pub struct Signer {
```

Comment:
I would probably think of renaming this struct into something like `SignedApi` or `TxApi` as its main purpose is executing signed transactions whereas the `api()` func could be renamed into something `unsigned()`. In that case any client code would look like
```rust
let signed_api: SignedApi = ...;

let source = signed_api.source();

let gprog = signed_api.unsigned().gprog(some_prog_id);

```

Reply:
I agree that current names are a little bit confusing.

Also, to me, this usage looks weird:

```rust
let signer = Api::new(...).signer(...);
let api = signer.api();
```

@clearloop What do you think about renaming? Let's choose the most appropriate names but in separate PR.

Reply:
to be considered in future